tests/console/snapper_cleanup_timeline.pm

```diff
@@ -36,36 +36,48 @@ sub convert2numeric {
     return $str2convert;
 }
 
+# returns the percentage % of used disk space.
 sub get_used_disk_space {
     my $used_space = script_output("df -h /  | awk \'NR==2 {print \$5}\'");
     return convert2numeric($used_space);
 }
 
+# returns the disk size number without units
+sub get_disk_size {
+    my $disk_size = script_output("df -h /  | awk \'NR==2 {print \$2}\'");
+    return convert2numeric($disk_size);
+}
+
 sub run {
     my $test_data = get_test_suite_data();
     record_info("Check quota", "Verify that the percentage of root filesystem quota is less than 50%");
     my $qgroup_space = script_output("btrfs qgroup show --sync --si / | grep \"1/0\" | awk \'{print \$3}\'");
     $qgroup_space = convert2numeric($qgroup_space);
     my $space_limit = $test_data->{snapper_config}->{SPACE_LIMIT};
-    my $disk_size   = script_output("df -h /  | awk \'NR==2 {print \$2}\'");
-    $disk_size = convert2numeric($disk_size);
+    my $disk_size   = get_disk_size();
```

Comment:
I was wondering about it and I read in a forum that it's better to keep as common practice, adding the parentheses even when not needed just in case there is a class or object with the same name.

Reply:
we don't use object or classes much in the repo, I guess it is because of that I was said that :) , it is a valid point. thanks for clarify it.